### PR TITLE
Health and Metrics APIs

### PR DESCRIPTION
**Pull request by ninlar**
Adding health and metrics APIs

### CODE REVIEW

File: Documentation/Operations/Operations_Metrics.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+# Metrics
+
+APIs related to querying tenant metrics.
+
+
+***
+
+## Authentication
+
+All endpoints referenced in this documentation require authenticated access. Authorization header must be set to the access token you retrieved after a successful authentication request.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Metrics.md:14]: Change to "You must set the Authorization header to the access token you retrieved from a successful authentication request."

File: Documentation/Operations/Operations_Metrics.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+`Authorization: Bearer <token>`
+
+Requests made without an access token or an invalid/expired token will fail with a 401 Unauthorized response.
+Requests made with an access token which does not have the correct permissions (see security subsection on every endpoint) will fail with a 403 Forbidden.
+Read [OCS Authentication documentation](https://github.com/osisoft/OSI-Samples-OCS/tree/master/basic_samples/Authentication) on how to authenticate against OCS with the various clients and receive an access token in response.
```
Comment (osimarkrogers) [inline Documentation/Operations/Operations_Metrics.md:20]: Change "on how to" to "to learn how to"

File: Documentation/Operations/Operations_Health.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+# Health
+
+APIs related to querying OCS Service health states.
+
+
+***
+
+## Authentication
+
+All endpoints referenced in this documentation require authenticated access. Authorization header must be set to the access token you retrieved after a successful authentication request.
```
[the remark this answers is not in the document]
Comment (ninlar) [reply Documentation/Operations/Operations_Health.md]: Resolved

File: Documentation/Operations/Operations_Metrics.md (hunk 1 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Metrics.md]: Resolved

File: Documentation/Operations/Operations_Health.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+`Authorization: Bearer <token>`
+
+Requests made without an access token or an invalid/expired token will fail with a 401 Unauthorized response.
+Requests made with an access token which does not have the correct permissions (see security subsection on every endpoint) will fail with a 403 Forbidden.
+Read [OCS Authentication documentation](https://github.com/osisoft/OSI-Samples-OCS/tree/master/basic_samples/Authentication) on how to authenticate against OCS with the various clients and receive an access token in response.
```
[the remark this answers is not in the document]
Comment (ninlar) [reply Documentation/Operations/Operations_Health.md]: Resolved

File: Documentation/Operations/Operations_Metrics.md (hunk 2 above)
Comment (ninlar) [reply Documentation/Operations/Operations_Metrics.md]: Resolved

**Pull request merged**